ops/pebble.py

```diff
@@ -832,6 +841,17 @@ def stop_services(
         """
         return self._services_action('stop', services, timeout, delay)
 
+    def restart_services(
+        self, services: typing.List[str], timeout: float = 30.0, delay: float = 0.1,
+    ) -> ChangeID:
+        """Restart services by name and wait (poll) for them to be started.
+
+        Raises ChangeError if one or more of the services didn't stop/start. If
+        timeout is 0, submit the action but don't wait; just return the change
+        ID immediately.
+        """
```

Comment:
What should happen if the list of services is empty? eg `restart_services([])` does it restart all, is it an error?